<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Categorias</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imagenes/logoLibreria.svg</normaloff>../imagenes/logoLibreria.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QTableWidget" name="tblWdgtUsuarios">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>791</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(0, 0, 0);
border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-10</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Categorias</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsertar">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>477</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>INSERTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>477</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>477</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCategoria">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Categoria</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstado">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spIdCategoria">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>550</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMensaje">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,10);
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReporte">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(50, 145, 165);
}
QPushButton{
	
	background-color: rgb(255, 138, 42);
	color: rgb(255,255,255)
}
QPushButton:hover{
background-color: rgb(255, 104, 3);

}</string>
    </property>
    <property name="text">
     <string>Generar Repote</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>901</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/fondo.svg</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>tblWdgtUsuarios</zorder>
   <zorder>label</zorder>
   <zorder>btnInsertar</zorder>
   <zorder>btnActualizar</zorder>
   <zorder>btnEliminar</zorder>
   <zorder>txtCategoria</zorder>
   <zorder>txtEstado</zorder>
   <zorder>spIdCategoria</zorder>
   <zorder>lblMensaje</zorder>
   <zorder>btnReporte</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
